harness/determined/common/experimental/determined.py

```diff
@@ -68,6 +68,7 @@ def create_experiment(
         self,
         config: Union[str, pathlib.Path, Dict],
         model_dir: Union[str, pathlib.Path],
```

Comment:
It looks like are currently using `model_dir` here, `model_context` in [common/api/experiment.py](https://github.com/rb-determined-ai/determined/blob/9eab0f7d18565a66f7b739b073d8bf14879e20c4/harness/determined/common/api/experiment.py#L124), and `--context` in the CLI to refer to the same idea in all three cases.

Up to you whether standardizing nomenclature is worth making a breaking change.

Reply:
all of `common` is an internal and therefore irrelevant, from a user-facing API perspective.

Yeah it's a bit of an unfortunate name but we're not going to break it over this.  Perhaps when we go from experimental to GA we could consider making that change at the same time.  Feel free to make a ticket for it and mark it as "breaking change", I think it's a good idea.

Reply:
Ok cool, thanks for explaining. That was my sense of what `common` was, but I wasn't sure.